otter/test/convergence/test_service.py

```diff
@@ -1366,6 +1374,41 @@ def plan(*args, **kwargs):
             perform_sequence(self.get_seq() + sequence, self._invoke(plan)),
             ConvergenceIterationStatus.Stop())
 
+    def test_launch_stack_config(self):
+        """Using a launch_stack launch config works."""
```

Comment:
This test checks launch_stack config with no steps. It will be nice to mention that in the docstring. Also, we may need to one more test case which returns steps that checks if they are executed. That can be another PR though :)